src/components/com_tjdashboard/site/assets/js/tjDashboardUI.js

```diff
@@ -61,9 +66,9 @@ var TJDashboardUI = {
 		});
 	},
 
-	initWidget : function(id){
+	initWidget : function(widget_data){
```

Comment:
@monikaatvowel Use camel case variables. Also, minify the js file and load the minified version.